source/clear-linux/guides/guides.rst

```diff
@@ -3,19 +3,16 @@
 Guides
 ######
 
-Our Guides: 
-
-*  Provide a critical, fundamental understanding of |CL| features
-*  Show you how to leverage the full feature set of |CL|
-*  Enhance your productivity when using |CL| 
-
-
-The following guides provide step-by-step instructions for tasks that come
-after completing the |CL| :ref:`installation <get-started>`.
+Use our guides to learn how to complete common tasks in |CL-ATTR| and use 

```

Comment:
I like this section better as a list of bullets. A previous issue tasked @intelkevinputnam and I with defining guides and distinguishing them from tutorials. We chose to use bullets because they're more efficient than prose--and more likely to be read. I recommend keeping bullets. I'm open to you revising, refining the language. 

Others, like @tpleavitt and @bd-dean, recently commented that they liked these pages as they are.